app/src/main/res/values-vi/strings.xml

```diff
@@ -62,7 +62,7 @@
     <string name="my_data_column_time">Thời gian</string>
     <string name="my_data_column_tuid">ID</string>
     <string name="my_data_column_rssi">RSSI</string>
-    <string name="about_title">Về ứng dụng</string>
+    <string name="about_title">V\u1ec1 \u1ee9ng d\u1ee5ng</string>
```

Comment:
tvl tole psat to musi bejt slusnej opruz 😃

Reply:
To dělá OneSky :)